shared/constants/network.ts

```diff
@@ -469,6 +481,11 @@ export const BUILT_IN_NETWORKS = {
     chainId: CHAIN_IDS.SEPOLIA,
     ticker: TEST_NETWORK_TICKER_MAP[NETWORK_TYPES.SEPOLIA],
   },
+  [NETWORK_TYPES.CONSENSYS_ZKEVM]: {
```

Comment:
I don't think we want this in the built in networks, but rather in the custom network list

Reply:
so in `FEATURED_RPCS`

Reply:
oh, I see that you've added to the default dropdown via multiple code changes... so this is a product decision